### DISCUSSION TITLE
Is there a way to adjust the MT and MD options when VCPKG compile?

### DISCUSSION THREAD

**Discussion by 1027663760** [microsoft/vcpkg#37347] [Q&A] [answered]
The command that can be available now:
.\vcpkg install zlib:x86-windows-static
.\vcpkg install zlib:x86-windows-static-md

I want this effect:
.\vcpkg install zlib:x86-windows-mt
.\vcpkg install zlib:x86-windows-md

I checked the document and did not find the relevant method

**Comment by dg0yt** [comment, marked as answer]
The relevant method: create your own triplet files. 
E.g. copy the x64 triplet files and change the architecture variable in those copies. Keep them in a separate dir and add integrate them as "overlay triplets".
https://learn.microsoft.com/en-us/vcpkg/concepts/triplets

**Reply by 1027663760** [reply]
Thanks!
After modifying this file, I successfully compiled the/mt version of the DLL file

vcpkg\triplets\x86-windows.cmake

set(VCPKG_CRT_LINKAGE dynamic) → set(VCPKG_CRT_LINKAGE static)

**Reply by dg0yt** [reply]
Note that changing the meaning of an existing triplet name may cause confusion the next time you ask for help.
